doc/changelog/02-specification-language/10167-orpat-mixfix.rst

```diff
@@ -0,0 +1,12 @@
+- Require parentheses around nested disjunctive patterns, so that pattern and
+  term syntax are consistent; match branch patterns no longer require
+  parentheses for notation at level 100 or more. Incompatibilities:
+
+  + in :n:`match p with (_, (0|1)) => ...` parentheses may no longer be
+    omitted around :n:`0|1`.
+  + notation :n:`(p | q)` now potentially clashes with core pattern syntax,
```

Comment:
That's not very important but here the `:n:` quoting context is not really useful given there are no quoted `@token`. The `:g:` context (for Gallina) could be preferred.